python/ray/experimental/gpu_object_manager/gpu_object_store.py

```diff
@@ -142,3 +154,19 @@ def remove_gpu_object(self, obj_id: str):
         del self.gpu_object_store[obj_id]
         if obj_id in self.primary_gpu_object_ids:
             self.primary_gpu_object_ids.remove(obj_id)
+
+    def decrement_num_readers(self, obj_id: str):
+        """
+        Decrement the number of readers for a GPU object. This function should
+        only be called by the receiver actor.
```

Comment:
Same here about describing behavior instead of intended usage.

Should also say something about when this will remove the object.

Reply:
I think it would also be cleaner to not introduce new functions, but squash the logic for is_primary_copy and decrement_num_readers into `get_object`.

Then we can also reuse the new ref counting logic to handle primary copies (i.e. set their ref count to -1 for infinity).